packages/pwa/app/components/_app/index.jsx

```diff
@@ -257,6 +257,19 @@ App.getProps = async ({api}) => {
         }
     })
 
+    if (rootCategory.isError) {
+        const message =
+            rootCategory.title === 'Unsupported Locale'
+                ? `
+                
+🚫 This page isn’t working.
+It looks like the locale ‘en-GB’ hasn’t been set up, yet. 
```

Comment:
To locale short code here is a variable. You need to swap it out with whatever the local is being requests.

Reply:
This should also be localized.

Reply:
Fixed, thanks!